recipes/flask-babelex/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "flask-babelex" %}
+{% set camelName = "Flask-BabelEx" %}
```

Comment:
Something to consider is just make this the name and add `.lower()` when you don't want CamelCase.